build-tools/packages/build-cli/src/commands/typetests.ts

```diff
@@ -112,20 +112,46 @@ If targeting prerelease versions, skipping versions, or using skipping some alte
 			}
 
 			if (this.flags.normalize) {
-				json.typeValidation = {
-					disabled:
-						json.typeValidation?.disabled === true
-							? true
-							: defaultTypeValidationConfig.disabled,
-					broken: json.typeValidation?.broken ?? defaultTypeValidationConfig.broken,
-					entrypoint:
-						json.typeValidation?.entrypoint ?? defaultTypeValidationConfig.entrypoint,
-				};
+				json.typeValidation = normalizeConfig(json.typeValidation);
```

Comment:
This change has the effect that if you disable type tests, then reenable them, it forgets the other settings.

This could be undesirable.

I think you can get most of the same effects, if you preserved existing settings when disabled, but have `normalized.broken` be omitted instead of included when empty.

That said, we don't enable or disable type tests very often, so I don't think it matters much so I'm fine with it as is.